SwiftUIHomeworks/SwiftUIHomeworks/ContentView.swift

```diff
@@ -7,20 +7,222 @@
+  
+  private enum Constants {
+    static let settingsTitleText = "Настройки"
+    static let personPlaceholderImageName = "person.fill"
+    static let nameText = "Aleksandr Dorofeev"
+    static let settingsDescriptionText = "Apple ID, iCloud, контент и покупки"
+    static let suggestionsAppleIDText = "Предложения Apple ID"
+    static let completionSettings = "Завершение настройки"
+    static let smallCircleWarnImageName = "1.circle.fill"
+    static let rightChevronImageName = "chevron.right"
+    static let aviaModeToggleName = "Авиарежим"
+    static let airplaneImageName = "airplane"
+    static let wifiTextTitle = "Wi-Fi"
+    static let wifiImageName = "wifi"
+    static let connectedText = "Подключено"
+    static let notConnectedText = "Не подключено"
+    static let bluetoothTitle = "Bluetooth"
+    static let bluetoothImageName = "bluetooth"
+    static let mobileTitle = "Мобильная связь"
+    static let mobileWaveImageName = "antenna.radiowaves.left.and.right"
+    static let modemTitle = "Режим модема"
+    static let modemImageName = "personalhotspot"
+    static let vpnTitle = "VPN"
+    static let vpnImageName = "checkmark.icloud.fill"
+    static let onText = "Вкл."
+    static let offText = "Выкл."
+    static let emptyString = ""
+    static let imageWidthFrameValue: CGFloat = 30
+    static let imageHeightFrameValue: CGFloat = 30
+    static let cornerRadiusValue: CGFloat = 5
+    static let settingsTextFontSizeValue: CGFloat = 12
+    static let vStackSpacingValue: CGFloat = 20
+    static let cornerRadiusForImageValue: CGFloat = 30
+    static let frameForImageValue: CGFloat = 60
+    static let zeroIntValue = 0
+    static let oneIntValue = 1
+  }
+  
+  // MARK: - Public properties
+  
+  var body: some View {
+    VStack {
+      NavigationStack {
+        VStack(spacing: Constants.vStackSpacingValue) {
+          Form {
+            Section {
+              createUserInfoSectionView()
+              createSuggestionsAppleIDSectionView()
+            }
+            Section {
+              createCompletionSettingSectionView()
+            }
+            Section {
+              createAviaModeSectionView()
+              createWiFiSectionView()
+              createBluetoothSectionView()
+              createMobileSectionView()
+              createModemSectionView()
+              createVpnSectionView()
+            }
+          }
+          .navigationTitle(Constants.settingsTitleText)
         }
-        .padding()
+        .searchable(text: $isSearchingText)
+      }
     }
+  }
+  
+  // MARK: - Private properties
+  
+  @State private var isOn = false
+  @State private var isWiFiSelection = Constants.zeroIntValue
+  @State private var isBluetoothSelection = Constants.zeroIntValue
+  @State private var isVPNSelection = Constants.zeroIntValue
+  @State private var isSearchingText = Constants.emptyString
+  
+  // MARK: - Private methods
+  
+  fileprivate func createSuggestionsAppleIDSectionView() -> some View {
```

Comment:
```suggestion
  private var suggestionsAppleIDSectionView:  some View {
```
так же везде поправь